src/components/pages/BoatDetails.jsx

```diff
@@ -1,7 +1,92 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getBoatDetails } from '../../redux/boatDetails/boatDetailsSlice';
 
-const BoatDetails = () => (
-  <div>Boat Details</div>
-);
+const BoatDetails = () => {
+  const boatDetailsState = useSelector((state) => state.boatDetails);
+  const dispatch = useDispatch();
+  const idBoat = useParams();
+
+  const boat = boatDetailsState.boatDetails;
+
+  const boatName = boat.name;
```

Comment:
I noticed that you assigned a value for the name, but not the others, it might be easier to just destructure it if that was your plan like this: const { name, picture, description, price...(and the others) } = boat, or just use boat.name, to be consistent.